Test/git-issues/git-issue1495.dfy

```diff
@@ -0,0 +1,95 @@
+// RUN: %dafny /compile:0 /rprint:"%t.rprint" "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+include "../libraries/src/Wrappers.dfy"
+import opened Wrappers
+
+datatype Bar = Bar(i: string)
+function method ParseBar(s: string): Result<Bar, string> {
+   Success(Bar(s))
+}
+
+class Foo {
+  const bar: Bar
+
+  constructor (
+    barLike: string
+  )
```

Comment:
This style may be useful if there are lots of parameters, but since there's only one, I suggest putting it on the same line.